package.json

```diff
@@ -32,6 +32,7 @@
     "@types/react": "^16.0.0",
     "@types/react-dom": "^15.5.2",
     "jquery": "^2.2.0",
+    "nbdime": "^0.4.0",
```

Comment:
Nice to see this PR, but I think this line should be removed (the dependency goes the other way).